ql/src/java/org/apache/hadoop/hive/ql/exec/MoveTask.java

```diff
@@ -317,6 +320,36 @@ public int execute() {
     }
 
     try (LocalTableLock lock = acquireLockForFileMove(work.getLoadTableWork())) {
+      String storageHandlerClass = null;
+      Properties commitProperties = null;
+      boolean overwrite = false;
+
+      if (work.getLoadTableWork() != null) {
+        // Get the info from the table data
+        TableDesc tableDesc = work.getLoadTableWork().getTable();
+        storageHandlerClass = tableDesc.getProperties().getProperty(
```

Comment:
Can `tableDesc.getProperties()` be null?

Reply:
I think it should not be `null`, otherwise we will have a `NullPointerException` in `EXPLAIN ...` commands: https://github.com/apache/hive/blob/a97448f84167e4e8c3615908556fe2e4163a43ca/ql/src/java/org/apache/hadoop/hive/ql/plan/TableDesc.java#L165-L168

Reply:
Makes sense

Reply:
Did not change as discussed